nilearn/plotting/displays.py

```diff
@@ -703,6 +713,27 @@ def add_edges(self, img, color='r'):
 
         plt.draw_if_interactive()
 
+    def add_markers(self, marker_coords, marker_color, marker_size, **kwargs):
+        """Plot markers"""
+        defaults = {'marker': 'o',
+                    'zorder': 1000}
+        for k, v in defaults.items():
+            kwargs.setdefault(k, v)
+
+        for display_ax in self.axes.values():
+            direction = display_ax.direction
+            index = _get_index_from_direction(direction)
+            # coord = display_ax.coord[index]
+            coord = display_ax.coord
+            marker_coords_2d, third_d = _coords_3d_to_2d(
+                marker_coords, direction, return_direction=True)
+            # XXX: should we keep this heuristic?
```

Comment:
What does the heuristic do? I don't even understand (it would call for a line of comment).